### DISCUSSION TITLE
Progress Bar inverted

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1112] [Proposals]
Componente global + código de inversión.

Ejemplo: en lugar de llenar la barra de progreso y mostrar cuánto Internet se usó, el comportamiento debería disminuir y mostrar cuánto Internet aún tiene disponible el usuario.

@yceballost ¿Sería global o extendido?

https://rebrand.ly/5p00e5i

**Comment by yceballost** [comment]
Hola Anne!

Creemos que tiene sentido que la progress bar pueda tener este comportamiento invertido para casos como el que comentas

Lo añadimos al backlog! #1116

**Reply by annefranca** [reply]
Olá @yceballost :)

Adicionei o comportamento "inverted" na Branch, segue o link: 
[#1116 - Progress Bar Inverted](https://www.figma.com/file/3IO1EsGDeBRiFdF23mrvuj/branch/okNmAEPy7ln9PJ4Z3mHX7m/%F0%9F%94%B8-ProgressBar-Specs?node-id=306%3A1704&t=aitg0jfYIOMYHijN-1)
